plugins/filter_aws/aws.c

```diff
@@ -1048,6 +1116,14 @@ static struct flb_config_map config_map[] = {
      "if both tags_include and tags_exclude are specified, configuration is invalid"
      " and plugin fails"
     },
+    {
+     FLB_CONFIG_MAP_INT, "retry_interval_s", "300",
+     /* for the time being, the only group which has retries is ec2 tags */
+     /* therefore configuration immediately sets this value just for ec2 tags */
+     0, FLB_TRUE, offsetof(struct flb_filter_aws,
+             metadata_groups[FLB_FILTER_AWS_METADATA_GROUP_TAGS].retry_required_interval),
+     "Defines minimum duration between retries for fetching EC2 instance tags"
```

Comment:
update description command and field so that its for all metadata?

Reply:
Well, right now, retries are specifically for the EC2 tags. In the future, if we add retries for more groups, it will be fine to just change the description without affecting the type and config name.

Should we change the description right now?

Reply:
Why not just make it this retry setting affect all metadata right now? It would be a small code change, and then you can add other options in the future. This way the config option name which makes it sound like it applies to all metadata makes sense from the beginning.

Reply:
Bump.

Reply:
It applies to all metadata groups after my changes in 846182780dbad63fcdd8034f6675d7c74b3f18ba.